assignments/1. hello-world/external/index.html

```diff
@@ -4,7 +4,10 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <title>Internal</title>
+    <title>External</title>
   </head>
-  <body></body>
+  <body>
+    <script src="assets/stylesheets/script.js">
```

Comment:
When using a external file you should use './' infront of the file name. For example "./assets..."

Reply:
resolved